mesh.py

```diff
@@ -44,24 +62,44 @@ def flux_add(self, cell, distance):
     def flux_clear(self):
         self._flux[:][:][:] = 0
 
-    def display_flux(self, index):
-        plotter.plot_heat_map(self._flux, index)
-    
+    def display_flux(self):
+        print self._flux
+
+    def display_materials(self):
+        print self._cell_materials
+
     def get_flux(self):
         return self._flux
     
-    def get_cell_max(self, position):
-        cell_number = self.get_cell(position)
+    def get_cell_max(self, position, direction=[0,0,0]):
+        cell_number = self.get_cell(position, direction)
         maxes = dict()
-        for num, axis in zip(xrange(3), ['x', 'y', 'z']):
-            maxes.update({axis:((cell_number[num] + 1) * \
+        for i, axis in enumerate(['x', 'y', 'z']):
+            maxes.update({axis:((cell_number[i] + 1) * \
                     self._delta_axes[axis] + self._boundary_mins[axis])})
         return maxes
 
-    def get_cell_min(self, position):
-        cell_number = self.get_cell(position)
+    def get_cell_min(self, position, direction=[0,0,0]):
```

Comment:
In the get_cell function, direction is used on a check in an if statement. I was under the impression that if get_cell wasn't passed a direction (like when the cells are being filled with a material: no direction exists), it would throw an error when asked to evaluate that if statement. If that's incorrect I can remove the default direction value.